docs/topics/plugin-icon-legend.adoc

```diff
@@ -5,17 +5,17 @@
 [id='plugin-icon-legend_{context}']
 = About {ProductShortName} issue icons
 
-In the Issue Explorer and file editors, {ProductShortName} issues use an icon to indicate their severity level and status. The following table describes the meaning of the various icons.
+{ProductShortName} displays icons in *Issue Explorer* and file editors to indicate the severity and status of issues related to migration. The table below describes the meaning of these icons.
 
 .{ProductShortName} issue icons
 [cols="20%a,80%",options="header",]
 |====
 |Icon |Description
 |image::error.png[Mandatory] |The issue is mandatory to fix for a successful migration.
 |image::info.gif[Optional] |The issue is optional to fix for migration.
-|image::warning.png[Warning] |The issue may potentially be an issue during migration.
-|image::fixedIssue.gif[Resolved] |The issue has been resolved.
-|image::stale_issue.gif[Stale] |The issue is stale because the code marked as an issue has been modified since the last time that {ProductShortName} was run on the project.
+|image::warning.png[Warning] |The issue might be an issue during migration.
+|image::fixedIssue.gif[Resolved] |The issue was resolved.
+|image::stale_issue.gif[Stale] |The issue is stale. The code marked as an issue was modified since the last time that {ProductShortName} identified it as an issue.
 |image::quickfix_error.png[Mandatory with quick fix] |A quick fix is available for this issue, which is mandatory to fix for a successful migration.
```

Comment:
I just noticed that the quick fix icons are already in the section about quick fixes. So they can be removed from here.